pkgs/top-level/python-packages.nix

```diff
@@ -2477,7 +2477,17 @@ in modules // {
     };
   };
 
-  boto3 = buildPythonPackage rec {
+  boto3 = let
+    # Version >=1.3,<1.4 is required by boto3.
+    botocore-1_3 = self.botocore.override rec {
+      name = "botocore-${version}";
+      version = "1.3.30";
+      src = pkgs.fetchurl {
+        url = "https://pypi.python.org/packages/6c/79/5d3abfb9fffe4a8cb0a113f088c1012743cf40db805aaac625b2e3cf2111/botocore-1.3.30.tar.gz";
```

Comment:
I see that. I'd prefer to just have a generic "buildPypiUrl" function (actually I have a `pypiPackage` function in my company's nix code for this reason). The reason I'm a bit ambivalent is that it's not possible to "accidentally" get out of sync because of the SHA. But it's fine.